src/components/pagination/daikin-pagination.ts

```diff
@@ -0,0 +1,232 @@
+  [
+    "border-0",
+    "no-underline",
+    "flex",
+    "relative",
+    "items-center",
+    "justify-center",
+    "min-w-12",
+    "min-h-12",
+    "font-daikinSerif",
+    "text-base",
+    "not-italic",
+    "leading-6",
+    "focus-visible:outline",
+    "focus-visible:outline-2",
+    "focus-visible:outline-offset-1",
+    "focus-visible:outline-[#0081C0]",
+  ],
+  {
+    variants: {
+      active: {
+        true: [
+          "text-daikinBlue-500",
+          "focus-visible:text-daikinBlue-500",
+          "font-bold",
+          "hover:text-[#0081C0]",
+          "hover:bg-[#DDF3FC]",
+          "active:text-[#00689A]",
+          "active:bg-[#BBE7F9]",
+          "after:bg-daikinBlue-500",
+          "after:content-['']",
+          "after:h-1",
+          "after:absolute",
+          "after:inset-0",
+          "after:top-auto",
+        ],
+        false: [
+          "text-inherit",
+          "font-normal",
+          "text-daikinNeutral-700",
+          "active:bg-daikinNeutral-100",
+          "active:text-daikinNeutral-800",
+          "hover:bg-[#F2F2F2]",
+          "hover:text-[#515151]",
+        ],
+      },
+    },
+  }
+);
+
+const cvaEllipsis = cva([
+  "text-inherit",
+  "border-0",
+  "no-underline",
+  "flex",
+  "items-center",
+  "justify-center",
+  "w-12",
+  "h-12",
+  "font-daikinSerif",
+  "text-base",
+  "not-italic",
+  "font-normal",
+  "leading-6",
+]);
+
+const cvaChevronButton = cva([
+  "text-inherit",
+  "border-0",
+  "no-underline",
+  "flex",
+  "items-center",
+  "justify-center",
+  "w-12",
+  "h-12",
+  "text-[#515151]",
+  "font-daikinSerif",
+  "text-base",
+  "not-italic",
+  "font-normal",
+  "leading-6",
+  "enabled:hover:bg-[#F2F2F2]",
+  "enabled:hover:text-daikinNeutral-800",
+  "enabled:active:bg-daikinNeutral-100",
+  "enabled:active:text-daikinNeutral-900",
+  "disabled:!text-daikinNeutral-400",
+  "focus-visible:outline",
+  "focus-visible:outline-2",
+  "focus-visible:outline-offset-1",
+  "focus-visible:outline-[#0081C0]",
+]);
+
+/**
+ * The pagination component is used to navigate through a list of items that are divided into multiple pages.
+ *
+ * @fires change - Emitted when the pagination current is changed.
+ *
+ * @example
+ *
+ * ```html
+ * <!-- You can create a pagination just specify the max pages and how many pages should be show. -->
+ * <daikin-pagination total="5" window="5">
+ * </daikin-pagination>
+ * ```
+ */
+@customElement("daikin-pagination")
+export class DaikinPagination extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: inline-flex;
+    }
+  `;
+
+  /**
+   * The current page number, starting at 1.
+   * Must be greater than 0 and less than or equal to `this.total`.
+   */
+  @property({ type: Number, reflect: true })
+  current = 1;
+
+  /**
+   * The number of pages.
+   * Must be greater than 0.
+   */
+  @property({ type: Number, reflect: true })
+  total = 5;
```

Comment:
nit. I don't think that we should make the default values seem like they have meaning.

```suggestion
  total = 1;
```